packages/graphics/lcms2/package.mk

```diff
@@ -2,10 +2,11 @@
 # Copyright (C) 2020-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="lcms2"
-PKG_VERSION="2.9"
-PKG_SHA256="8e23a09dc81af856db37941a4ea26acdf6a45b0281ec5b7ee94b5a4e9f7afbf7"
-PKG_LICENSE="MIT"
+PKG_VERSION="2.11"
+PKG_SHA256="dc49b9c8e4d7cdff376040571a722902b682a795bf92985a85b48854c270772e"
+PKG_LICENSE="MIT/GPLv3"
 PKG_SITE="http://www.littlecms.com"
-PKG_URL="https://github.com/mm2/Little-CMS/archive/lcms${PKG_VERSION}.tar.gz"
+PKG_URL="$SOURCEFORGE_SRC/lcms/lcms/${PKG_VERSION}/${PKG_NAME}-${PKG_VERSION}.tar.gz"
```

Comment:
why change to sourceforge?

Reply:
@mm2 moved the release to SF. https://github.com/mm2/Little-CMS/releases/tag/2.11

Reply:
meh, if it works I guess it's fine